### PR TITLE
Erc7412 minimum credit check

### PR DESCRIPTION
**Pull request by dbeal-eth**

### CODE REVIEW

File: markets/perps-market/storage.dump.json [hunk 1]
```diff
@@ -228,46 +228,53 @@
           "slot": "6",
           "offset": 0
         },
+        {
+          "type": "uint256",
+          "name": "currentLiquidationReward",
```
Comment (noisekit) [inline markets/perps-market/storage.dump.json:233]: What are these storage slots changes?

Comment (dbeal-eth) [reply markets/perps-market/storage.dump.json:233]: Sorry my local setup was messed up and produced these. Should be removed

File: markets/perps-market/cannonfile.toml [hunk 2]
```diff
@@ -145,6 +148,7 @@ func = "initializeFactory"
 args = [
     "<%= imports.synthetix.contracts.CoreProxy.address %>",
     "<%= imports.spotMarket.contracts.SpotMarketProxy.address %>",
+    "<%= settings.minDelegationTime %>",
```
Comment (noisekit) [inline markets/perps-market/cannonfile.toml:151]: I don't see any changes in contract. Is this necessary?

Comment (dbeal-eth) [reply markets/perps-market/cannonfile.toml]: reverted external problematic change after discussion

**Review by noisekit — APPROVED** [review]
Storage c

**Pull request merged**